<launch>

  <node pkg="controller" name="controller_node" type="controller_node" output="screen"/>

</launch>